--- a/src/hbbr.rs
+++ b/src/hbbr.rs
@@ -8,7 +8,13 @@ mod version;
 
 fn main() -> ResultType<()> {
     let _logger = Logger::try_with_env_or_str("info")?
-        .log_to_stdout()
+        .log_to_file(
+            FileSpec::default()
+                .directory("/var/run/rustdesk-server")
+                .basename("hbbr")
+                .suppress_timestamp()
+                .suffix("log")
+        )
         .format(opt_format)
         .write_mode(WriteMode::Async)
         .start()?;
--- a/src/main.rs
+++ b/src/main.rs
@@ -9,7 +9,13 @@ const RMEM: usize = 0;
 
 fn main() -> ResultType<()> {
     let _logger = Logger::try_with_env_or_str("info")?
-        .log_to_stdout()
+        .log_to_file(
+            FileSpec::default()
+                .directory("/var/run/rustdesk-server")
+                .basename("hbbs")
+                .suffix("log")
+                .suppress_timestamp()
+        )
         .format(opt_format)
         .write_mode(WriteMode::Async)
         .start()?;
